doc/release/1.13.0-notes.rst

```diff
@@ -132,6 +138,7 @@ This means that, e.g., it is now possible to do the following:
     [array([[ 1. ,  1. , -0.5], [ 1. ,  1. , -0.5]]), 
      array([[ 2. ,  2. ,  2. ], [ 2. ,  1.7,  0.5]])]
 
+
```

Comment:
While I'm there, should it be using tildes or hyphens as a vertical line? The entries in the new features section aren't consistent.